amethyst_controls/src/bundles.rs

```diff
@@ -8,10 +8,27 @@ use amethyst_core::{
 use super::*;
 
 /// The bundle that creates a flying movement system.
-/// Note: Will not actually create a moving entity. It will only register the needed resources and systems.
-/// The generic parameters A and B are the ones used in InputHandler<A,B>.
-/// You might want to add "fly_movement" and "free_rotation" as dependencies of the TransformSystem.
+///
+/// Note: Will not actually create a moving entity. It will only register the needed resources and
+/// systems. The generic parameters `A` and `B` are the ones used in `InputHandler<A,B>`.
+///
+/// You might want to add `"fly_movement"` and `"free_rotation"` as dependencies of the
+/// `TransformSystem` in order to apply changes made by these systems in the same frame.
 /// Adding this bundle will grab the mouse, hide it and keep it centered.
+///
+/// # Generic parameters
+///
+/// * `A`: This is the key the `InputHandler` is using for axes. Often, this is a `String`.
+/// * `B`: This is the key the `InputHandler` is using for actions. Often, this is a `String`.
+///
+/// # Systems
+///
+/// This bundle adds the following systems:
+///
+/// * `FlyMovementSystem`
+/// * `FlyRotationSystem`
+/// * `MouseFocusUpdateSystem`
+/// * `CursorHideSystem`
```

Comment:
Add link to systems?

Reply:
Yes, I'd also like to see more cross referencing but I've also seen it break in some places due to no standard way of linking and re-exports. Maybe that's fixed by now, I'm probably not up to date here.